src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,55 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import mate.academy.lib.Inject;
+
+@Entity
+@Inject

```

Comment:
The `@Inject` annotation should not be used on this class. This annotation is intended for dependency injection in service classes.